dashboards/confluent-cloud/confluent-cloud.json

```
@@ -1,622 +1,633 @@
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT rate(sum(confluent_kafka_server_request_bytes), 1 second) FROM Metric WHERE\n`kafka_id` in ({{kafka_id}}) FACET type LIMIT MAX TIMESERIES"
               }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "units": {
+              "unit": "BYTES_PER_SECOND"
+            },
+            "yAxisLeft": {
+              "zero": true
             }
+          }
+        },
+        {
+          "title": "Response Bytes (bytes/second) Per Kafka Protocol Request Type",
+          "layout": {
+            "column": 9,
+            "row": 16,
+            "width": 4,
+            "height": 3
           },
-          {
-            "title": "Broker Respone Size Average",
-            "layout": {
-              "column": 9,
-              "row": 16,
-              "width": 4,
-              "height": 3
-            },
-            "linkedEntityGuids": null,
-            "visualization": {
-              "id": "viz.line"
-            },
-            "rawConfiguration": {
-              "facet": {
-                "showOtherSeries": false
-              },
-              "legend": {
-                "enabled": true
-              },
-              "nrqlQueries": [
-                {
-                  "accountId": 0,
-                  "query": "SELECT latest(kafka.brokers.response_size_avg) FROM Metric TIMESERIES FACET `broker` LIMIT 1000"
-                }
-              ],
-              "platformOptions": {
-                "ignoreTimeRange": false
-              },
-              "units": {
-                "unit": "BYTES"
-              },
-              "yAxisLeft": {
-                "zero": true
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT rate(sum(confluent_kafka_server_response_bytes), 1 second) FROM Metric WHERE\n`kafka_id` in ({{kafka_id}}) FACET type LIMIT MAX TIMESERIES"
               }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "units": {
+              "unit": "BYTES_PER_SECOND"
+            },
+            "yAxisLeft": {
+              "zero": true
             }
           }
-        ]
-      }
-    ],
-    "variables": []
-  }
+        }
+      ]
+    }
+  ],
+  "variables": [
+    {
+      "name": "kafka_id",
+      "items": null,
+      "defaultValues": [],
+      "nrqlQuery": {
+        "accountId": 0,
```

Hey @4demos, it looks like this is the culprit. If we convert it over to `accountIds` version it should work just fine.
```suggestion
        "accountIds": [],
```

---

good catch! 
still not sure why in most cases we replace `"accountIds": [...]` with `"accountId": 0` but here we should keep "accountIds"

---

Agreed! Did the dashboard cleanup script adjust it to `"accountId": 0`?

---

`sanitize-dashboard` doesn't set "accountId": 0, just removes the actual account. But instructions https://developer.newrelic.com/contribute-to-quickstarts/build-a-quickstart/add-dashboard/ tell submitters: "Set your accountId to 0". If it's not a requirement any longer and `"accountIds": []` is fine, may be it's time to update the docs?

---

@4demos, thank you! I'll get the docs updated